src/main/webapp/CreateAnEvent.html

```diff
@@ -15,7 +15,7 @@
   <body onload="onload()">
     <div class="navbar">
       <p class="navbar-text" onclick="window.location.href = 'map.html'">Social Map</p>
-      <p class="navbar-text" onclick="window.location.href = 'nearme.html'">Near Me</p>
+      <p class="navbar-text" onclick="window.location.href = 'nearyou.html'">Near You</p>
```

Comment:
I don't see a file called `nearyou.html` in the repository, is it supposed to be added in this pull request?

Reply:
Sorry, I didn't realize that in renaming the file, git treated it as a new file. It is included in the commit below!

Reply:
No worries! In the future you may want to try `git mv nearme.html nearyou.html`--this way Git tracks the move (and it should produce a smaller diff in the logs and in the GitHub UI).